lighthouse-cli/index.js

```diff
@@ -113,9 +121,9 @@ if (cli.listAllAudits) {
 }
 
 if (cli.listTraceCategories) {
-  const categories = lighthouse.traceCategories;
+  const traceCategories = lighthouse.traceCategories;
 
-  process.stdout.write(JSON.stringify({traceCategories: categories}));
+  process.stdout.write(JSON.stringify({traceCategories}));
```

Comment:
I think it will be easier to interface with lighthouse-core in such situations rather than working through the CLI